tests/plugins/test_ignore/subdir1/test_load_sub1.py

```diff
@@ -0,0 +1,35 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""Import module"""
+from airflow.models.baseoperator import BaseOperator  # type: ignore
+from airflow.utils.decorators import apply_defaults  # type: ignore
+
+
+class Sub1TestLoadOperator(BaseOperator):
+    """
+    Test load operator
+    """
+    @apply_defaults
+    def __init__(
+            self,
+            *args,
+            **kwargs):
+        super(Sub1TestLoadOperator, self).__init__(*args, **kwargs)
+
+    def execute(self, context):
+        pass
```

Comment:
What is the purpose of those files? I think I don't see it tests

Reply:
These files were used to files that should not be ignored.
( `self.assertEqual(detected_files, should_not_ignore_files)` Line 87 of the test_plugin_ignore.py. )

But....
I fixed these files to be generated by test_plugin_ignore.py.